pkgs/applications/version-management/gitoxide/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "gitoxide";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "Byron";
+    repo = "gitoxide";
+    rev = "v${version}";
+    sha256 = "0xpic9jx7nrxi5d8lqch2vxpvipx994d717c4n0kgr3ipyij1347";
+  };
+
+  cargoSha256 = "104lyfni75h1i30s2jlzf66sp1czfd9ywqz78kj4i7lfdf6fc4x9";
+
+  meta = with lib; {
+    description =
+      "A command-line application for interacting with git repositories";
+    homepage = "https://github.com/Byron/gitoxide";
+    # NOTE: the master branch is dual-licensed with APACHE but v0.3.0 is only MIT
+    license = licenses.mit;
+    maintainers = [ maintainers.syberant ];
+    platforms = platforms.all;
```

Comment:
You can omit this, as it's set by `buildRustPackage` to the appropriate setting for this case:

https://github.com/NixOS/nixpkgs/blob/6d0a85fe528e020eb7d8eb78d33ee87055c01c74/pkgs/build-support/rust/default.nix#L246

Reply:
Ah, good to know.